addons/godot-firebase/auth/auth.gd

```diff
@@ -368,18 +383,21 @@ func save_auth(auth : Dictionary) -> void:
         encrypted_file.store_line(to_json(auth))
         encrypted_file.close()
 
+
 # Function used to load the auth data file that has been stored locally
 # Note this does not work in HTML5 or UWP
 func load_auth() -> void:
 
     var encrypted_file = File.new()
     var err = encrypted_file.open_encrypted_with_pass("user://user.auth", File.READ, _config.apiKey)
     if err != OK:
-        Firebase._printerr("Error Opening File. Error Code: "+ err)
+        Firebase._printerr("Error Opening File. Error Code: " + err)
+        emit_signal("auth_request", "Error Opening File. Error Code: " + err, auth)
```

Comment:
One thing I'd like here is just a word saying it's an error opening the *Auth* file. We don't want to potentially confuse users of our library suggesting there might be one of their own files that's missing.

Reply:
Updated.